hazelcast/src/test/java/com/hazelcast/internal/nearcache/AbstractNearCacheBasicTest.java

```diff
@@ -335,7 +335,8 @@ public void testGetAsyncPopulatesNearCache() throws Exception {
 
         // generate Near Cache hits
         populateNearCacheAsync(context);
-
+        assertEquals("Not equals to expected Near Cache hit count",
```

Comment:
If you can assert the misses as well then `assertNearCacheStats(NearCacheTestContext<?, ?, ?, ?> context, long expectedOwnedEntryCount, long expectedHits, long expectedMisses)` would be a better match. Then you could also get rid of the `assertNearCacheSize()` call.